evap/evaluation/models_logging.py

```diff
@@ -271,6 +275,10 @@ def unlogged_fields(self):
         """Specify a list of field names so that these fields don't get logged."""
         return ['id', 'order']
 
+    @classmethod
```

Comment:
why is this a classmethod and not a regularly overridable method?

Reply:
I changed it to a `staticmethod` for now. I think it's a bit strange to override a static method with a class method (like `Evaluation` does), but in a way it makes sense. I don't know if this behavior is backed by the language standard though :man_shrugging: